### DISCUSSION TITLE
Ajouter un call to action à la page de garde du SILL pour demander aux visiteurs de créer un compte

### DISCUSSION THREAD

**Discussion by Sami-Hazgui** [codegouvfr/sill-web#468] [Ideas]
*   **Problème perçu:** Lorsque l'on se connecte sur le SILL pour la première fois, on ne comprend pas que s'inscrire en tant qu'utilisateur sur le SILL peut permettre d'échanger avec des référents.
*   **Contraintes pour régler ce problème:** La page de garde du SILL doit rester simple et lisible
*   **Solution proposée :** Ajouter un call to action en dessous de la phrase de présentation du SILL avec un lien hypertexte: "Catalogue de référence de logiciels libres recommandés pour l'ensemble de l'administration."

Call to action: "Vous êtes un agent public ? Créez un compte utilisateur pour accéder au catalogue et échanger avec des référents de logiciels libres"

Ajouter aussi le même call to action à la fin du "SILL en quelques mots" avec un lien hypertexte: " Créez un compte utilisateur pour échanger avec des référents SILL."

**Comment by guillermau** [comment]
Si cette fonctionnalité est mise en place, ce serait bien qu'elle soit activable selon les instances par config, je ne suis pas sur que le CNRS veuille cette fonctionnalité pour l'instant.
